python/lsst/ip/isr/isrMockLSST.py

```diff
@@ -120,6 +202,18 @@ def makeImage(self):
         """
         exposure = self.getExposure()
 
+        # Set up random number generators for consistency of components,
+        # no matter the group that are configured.
+        rngSky = np.random.RandomState(seed=self.config.rngSeed + 1)
```

Comment:
The default levels are defined in the config `setDefaults` above and I don't think they're too crazy.  However, the serial and parallel ramps are greatly exaggerated (which I don't think is a problem; it can highlight problems with our approach).  Also the crosstalk matrix is unrealistic which is also useful to highlight problems.